web/org.eclipse.xtend.web/.project

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>org.eclipse.xtend.web</name>
+	<comment/>
+	<projects/>
+	<natures>
+		<nature>org.eclipse.xtext.ui.shared.xtextNature</nature>
+		<nature>org.eclipse.jdt.core.javanature</nature>
+		<nature>org.springsource.ide.eclipse.gradle.core.nature</nature>
```

Comment:
are you sure we would want this nature? AFAIK the springsource.ide.eclipse.gradle has been retired, and the new buildship Gradle integration is the way to go forward..